### PR TITLE
create ubuntu channels with correct architecture and type

### PR DESCRIPTION
**Pull request by mcalmer**
## What does this PR change?

When fetching SUSE Manager Tools channel for Ubuntu from SCC we need to create them with the correct channel arch and the repo with the correct ContentSourceType.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: **internal**

- [x] **DONE**

## Test coverage
- No tests: **add explanation**
- Unit tests were added
- Cucumber tests were added

- [ ] **DONE**

## Links

Tracks # **add downstream PR, if any**

- [ ] **DONE**

## Changelogs

Copy the following sentence as a new comment if you don't need changelog entries:

> gitarro no changelog needed !!!

If the test `changelog_test`already run, then add another new comment with the following text:

> gitarro rerun changelog_test !!!

### CODE REVIEW

File: java/code/src/com/redhat/rhn/manager/content/MgrSyncUtils.java [hunk 1]
```diff
@@ -126,6 +126,9 @@ public static ChannelArch getChannelArch(PackageArch packageArch, String channel
         else if (arch.equals("ppc64")) {
             arch = "ppc";
         }
+        else if (arch.equals("amd64")) {
+            arch = "amd64-deb";
```
Comment (mbologna) [inline java/code/src/com/redhat/rhn/manager/content/MgrSyncUtils.java]: why not only `amd64`?

Comment (mcalmer) [reply java/code/src/com/redhat/rhn/manager/content/MgrSyncUtils.java]: amd64 is a RPM architecture in spacewalk code

Comment (mbologna) [reply java/code/src/com/redhat/rhn/manager/content/MgrSyncUtils.java]: Ok, SCC -> map "amd64" to "amd64-deb" and repo type "deb"

**Comment by mbologna** [comment]
Rubocop failure is unrelated to this PR

**Comment by mcalmer** [comment]
merged with https://github.com/uyuni-project/uyuni/pull/700

**Pull request closed without merging**